task-2/src/components/ErrorBoundary/index.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react'
+
+class ErrorBoundary extends Component {
+
+    constructor(props) {
```

Comment:
Consider using class properties instead of constructor.
https://michalzalecki.com/react-components-and-class-properties/

Reply:
Error boundary component now includes class properties.